kustomize/commands/edit/set/configmap.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2023 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package set
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+	"sigs.k8s.io/kustomize/api/ifc"
+	"sigs.k8s.io/kustomize/api/resource"
+	"sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kustomize/v5/commands/internal/kustfile"
+	"sigs.k8s.io/kustomize/kustomize/v5/commands/internal/util"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+func NewCmdSetConfigMap(
+	fSys filesys.FileSystem,
+	ldr ifc.KvLoader,
+	rf *resource.Factory,
+) *cobra.Command {
+	var flags util.ConfigMapSecretFlagsAndArgs
+	cmd := &cobra.Command{
+		Use:   "configmap NAME [--from-literal=key1=value1]",
```

Comment:
Should `Use` specify the optional `--namespace` and `--new-namespace` flags?

Reply:
Absolutely yes, just updated this to include these flags as optionals.